app/uploaders/image_uploader.rb

```diff
@@ -0,0 +1,51 @@
+# encoding: utf-8
+
+class ImageUploader < CarrierWave::Uploader::Base
+
+  # Include RMagick or MiniMagick support:
+  include CarrierWave::RMagick
+  # include CarrierWave::MiniMagick
+
+  # Choose what kind of storage to use for this uploader:
+  storage :file
+  # storage :fog
```

Comment:
@ianroberts131 For file uploads to work with a Rails app hosted on Heroku, the files will need to be stored on a different server by a different provider. (Heroku's architecture makes saving files to their server's filesystem impossible to do reliably).

Consider using Amazon's S3 service to store the uploaded images. These instructions will help get started using carrierwave to do this: https://github.com/carrierwaveuploader/carrierwave#using-amazon-s3

Your Rails app would still be hosted by Heroku. Its only the uploaded image files that would be stored on and served by Amazon S3.